opencensus/src/main/java/com/example/opencensus/Quickstart.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.opencensus;
+
+// [START monitoring_opencensus_metrics_quickstart]
+
+import com.google.common.collect.Lists;
+
+import io.opencensus.exporter.stats.stackdriver.StackdriverStatsExporter;
+import io.opencensus.stats.Aggregation;
+import io.opencensus.stats.BucketBoundaries;
+import io.opencensus.stats.Measure.MeasureLong;
+import io.opencensus.stats.Stats;
+import io.opencensus.stats.StatsRecorder;
+import io.opencensus.stats.View;
+import io.opencensus.stats.View.Name;
+import io.opencensus.stats.ViewManager;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.Random;
+
+public class Quickstart {
+  private static final int exportInterval = 60;
+  private static final MeasureLong latencyMs = MeasureLong.create(
+      "task_latency",
+      "The task latency in milliseconds",
+      "ms");
+  // Latency in buckets:
+  // [>=0ms, >=100ms, >=200ms, >=400ms, >=1s, >=2s, >=4s]
+  private static final BucketBoundaries boundaries = BucketBoundaries.create(
+      Lists.newArrayList(0., 100., 200., 400., 1000., 2000., 4000.));
+  private static final StatsRecorder statsRecorder = Stats.getStatsRecorder();
+
+  public static void main(String[] args) throws IOException, InterruptedException {
```

Comment:
There was a conversation about a similiar Stackdriver sample not using main because web frameworks (servlets, SpringBoot, etc) don't typically have an accessible main method. - maybe ask @SurferJeffAtGoogle if that's applicable here.

Reply:
Jeffrey, please weigh in. But from my perspective, the point of the sample is the code that's run within the `main` method. People who want to copy paste can just take that part and use it however they see fit. This sample in particular doesn't have any framework integration, so I just have the `exec:java` maven task configured in `pom.xml`.

Reply:
tbh, keep the code simple. If the purpose is to demonstrate OpenCensus, then `main` is fine. If the purpose is to show OpenCensus measuring a Servlet, then use a Servlet.

Keeping things simple & to the point is important. See:

https://github.com/saturnism/grpc-java-by-example
https://github.com/saturnism/opencensus-java-by-example

Reply:
The purpose of this particular sample is to demonstrate the simplest case of OpenCensus, so let's keep the `main` function.